spec/controllers/projects_controller_spec.rb

```diff
@@ -103,8 +103,8 @@
         # specifies that the Project created on the previous line
         # receives the :update_attributes message with whatever params are
         # submitted in the request.
-        expect_any_instance_of(Project).to receive(:update_attributes).with({'these' => 'params'})
-        put :update, {:id => project.id, :project => {'these' => 'params'}}
+        expect_any_instance_of(Project).to receive(:update_attributes).with({'title' => 'New Title'})
+        put :update, {:id => project.id, :project => {'title' => 'New Title'}}
```

Comment:
With the move to using strong parameters, this test was failing with the previously given parameter "this" with value "params". It seemed like using the name of an actual, permitted parameter for this test should be fine. I'm not sure that's totally correct, though, because  the comment says "whatever params" which implies _any_ params. But I also don't think we want to allow just _any_ param, right?

Reply:
Right, not any param.  I think the test was just to check if the update method existed and not that it failed.